networkx/algorithms/tree/tests/test_mst.py

```diff
@@ -706,3 +706,26 @@ def test_random_spanning_tree_additive_large():
     # Assert that p is greater than the significance level so that we do not
     # reject the null hypothesis
     assert not p < 0.05
+
+
+def test_random_spanning_tree_empty_graph():
+    """
+    Test on empty graph, should return an empty spanning tree
+    """
+
```

Comment:
No comments needed in tests -- using a good test function name is usually enough.

Do you want to test for the case with 1 node and a self-loop? You mentioned it in a comment. probably good to include it. Then that comment isn't needed too. :)

Reply:
Understood, will make these changes